scripts/bootstrap

```diff
@@ -5,4 +5,5 @@ cd $(dirname $0)/..
 
 apt-get update
 apt-get install -y python-pip python-dev
+apt-get install phantomjs
```

Comment:
you need the -y here so that it installs without prompting. Can add phantomjs to line 7.